stellarphot/settings/tests/test_custom_widgets.py

```diff
@@ -198,15 +198,22 @@ def test_edit_item_not_saved_after_cancel(self):
         # Should not save the item after clicking the No button
         camera = self.make_test_camera()
 
-        # Make an additional test camera so that we the result is the *second* camera
-        # in the list, not the first. If it is the first the outcome is the same
-        # whether the user was editing or making a new camera.
+        # Make an additional test camera so that the result is the *second* camera
+        # in the list, not the first. The workflow being test is that there are
+        # already two cameras, then the second one is selected (so that something is)
```

Comment:
Superfluous parenthesis (the closing parenthesis is on the next line):
```suggestion
        # already two cameras, then the second one is selected (so that something is
```